woocommerce/woocommerce-android
# Add toolbar for SelectPaymentMethodFragment

Part of #10570 

### Description

This PR introduces individual toolbars for SelectPaymentMethodFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Select payment screen(Order list -> Order detail -> Collect payment CTA) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="241" alt="Screenshot 2024-02-12 at 7 08 36 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/8422466b-2adc-4f7a-a859-d90330c0516c">
